src/simulator/world.py

```diff
@@ -170,7 +172,18 @@ def visualize(self) -> dict:
         """
         objects_info = dict()
         for oid in self.objects:
-            if self.objects[oid].shape is not None:
                 # TODO: to be replaced with proper handling of compound objects (if cond)
-                objects_info[oid] = self.objects[oid].visualize()
+            objects_info[oid] = self.objects[oid].visualize()
         return objects_info
+    
+    def dumb_all_owners_inf(self):
+        # dump visualization info for Owners between objects to the output json file
+        owners_info = {"Owners":{}}
```

Comment:
I think in the rest of the code, we are using lowercase for keys in our dictionaries (like 'shape' instead of 'Shape'). If this is the case, please make this one lowercase too.

Reply:
Also, despite its name, it seems that this dictionary actually stores the dependent objects, and not the owners. Am I right?

Reply:
In fact, this is where the owner writes to the file , so the visualizer understands the relationship between the objects (for example, the visualizer realizes that a simple object does not belong to anyone ,I wrote a proposal about it).

Reply:
In fact, this is where the owner writes to the file , so the visualizer understands the relationship between the objects (for example, the visualizer realizes that a simple object does not belong to anyone ,I wrote a proposal about it).

Reply:
@mehdi138r 
Please see the comment I left on your proposal. You're currently storing `"parent": [children]`, but perhaps it'd be better to store `"child": parent`. This way, you can store such a thing along with the shape info for each object (so not a separate map).